search-app/docs/api.md

```diff
@@ -805,6 +805,7 @@ __Example__
       "has_formats" : false,
```

Comment:
We should document this field in the ATOM response section.
Look for this in the documentation to see where it goes:
echo:hasSpatialSubsetting (collections only) | true if any of its associated services support spatial subsetting